app/models/user.rb

```diff
@@ -216,22 +216,10 @@ def miq_groups=(groups)
     self.current_group = groups.first if current_group.nil? || !groups.include?(current_group)
   end
 
-  def miq_group_ids
-    miq_groups.collect(&:id)
-  end
-
```

Comment:
miq_groups is a normal relation, so you should get miq_group_ids for free from Rails.  It must not have been a relation in the past, hence this method here.  This can go away, *and* you don't neet the pluck in the other locations.